Extensions/mass_deleter.js

```diff
@@ -320,6 +322,7 @@ XKit.extensions.mass_deleter = new Object({
 			XKit.extensions.mass_deleter.unlike_fail_count = 0;
 			XKit.extensions.mass_deleter.unlike_next_current = 0;
 			XKit.extensions.mass_deleter.unlike_likes_page++;
+			XKit.extensions.mass_deleter.unlike_next_page_url = $("#next_page_link").attr("href");
```

Comment:
This will start at the current page, rather then the beginning. Is this the expected behavior? should we try fetching /likes and grabbing the header instead?

Reply:
I assumed this is the expected behavior? It makes more sense like that IMO + it's an added functionality, if anything.